### DISCUSSION TITLE
Zenodo code

### DISCUSSION THREAD

**Discussion by alberto99** [maccallumlab/meld#141] [Q&A]
Hi Justin,
i believe you mentioned MELD would now automatically get new zenodo dois. I could not find that, but i did find the one you created for version 0.4.14 (10.5281/zenodo.5083678).

Should we have an updated one that is mentioned in the initial readme file?
THanks!
Al.

**Comment by jlmaccal** [comment]
I don't think it's setup to get them automatically. I think I did that manually.

**Reply by alberto99** [reply]
Thanks, i went ahead and did a link between GitHub and Zenodo, it should be accessible for all future versions:
https://doi.org/[10.5281/zenodo.7502226](https://doi.org/10.5281/zenodo.7502226)
